x-pack/legacy/plugins/apm/server/lib/helpers/es_client.ts

```diff
@@ -111,9 +110,11 @@ export function getESClient(req: Legacy.Request) {
         console.log(JSON.stringify(nextParams.body, null, 4));
       }
 
-      return cluster.callWithRequest(req, 'search', nextParams) as Promise<
-        AggregationSearchResponse<Hits, U>
-      >;
+      return (cluster.callWithRequest(
+        req,
+        'search',
+        nextParams
+      ) as unknown) as Promise<AggregationSearchResponse<Hits, U>>;
```

Comment:
This was needed because SearchResponse interface from elasticsearch is still with the old version of the hits.total, which is a number. And the AggregationSearchResponse now expects hits.total.value.